taskcluster/baseclient.py

```diff
@@ -256,6 +257,15 @@ def _hasCredentials(self):
             cred['accessToken']
         )
 
+    def _hasHawkHeader(self):
+        """ Return True, if hawk header is already given """
+        cred = self.options.get('credentials')
+        return (
+            cred and
+            'hawkHeader' in cred and
+            cred['hawkHeader']
+        )
```

Comment:
please rewrite this as a more clear structure.  Having two checks (in and truthy) are redundant in Python since we have the set-membership (`in`) feature.